cluster/addons/addon-manager/Makefile

```diff
@@ -15,14 +15,14 @@
 IMAGE=gcr.io/google-containers/kube-addon-manager
 ARCH?=amd64
 TEMP_DIR:=$(shell mktemp -d)
-VERSION=v6.2
+VERSION=v6.3
```

Comment:
we'll need to update
* cluster/images/hyperkube/static-pods/addon-manager-singlenode.json
* cluster/images/hyperkube/static-pods/addon-manager-multinode.json
* cluster/saltbase/salt/kube-addons/kube-addon-manager.yaml